spec/features/session_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.feature "Session", type: :feature do
+  scenario "A user can sign in with valid credentials" do
+    visit "session/new"
+    fill_in :email, with: "jay@gmail.com"
```

Comment:
perhaps put these steps into a web helper to avoid repetition further down the road?